docker-compose.yml

```
@@ -243,8 +243,8 @@ services:
             timeout: 30s
             retries: 3
         ports:
-            - "${RABBITMQ_PORT}:5672"
-            - "${RABBITMQ_ADMIN_PORT}:15672"
+            - "127.0.0.1:${RABBITMQ_PORT}:5672"
+            - "127.0.0.1:${RABBITMQ_ADMIN_PORT}:15672"
```

we want the admin port to be exposed so that we can monitor the status of the queue using the admin page 
```suggestion
            - "${RABBITMQ_ADMIN_PORT}:15672"
```

---

good point!